docker/README.md

```diff
@@ -51,13 +51,15 @@ Simple stand-alone usage:
 
 To link to another mysql container, for example:
 
-    docker run --rm --name newcentury_mysql -p 3306 \
+    docker run --rm --name mysql \
         -e HUGINN_MYSQL_DATABASE=huginn \
         -e HUGINN_MYSQL_USER=huginn \
         -e HUGINN_MYSQL_PASSWORD=somethingsecret \
         -e HUGINN_MYSQL_ROOT_PASSWORD=somethingevenmoresecret \
```

Comment:
It should actually be HUGGIN_ prefixed due to the logic here:

https://github.com/cantino/huginn/blob/master/docker/scripts/init#L29

This:

```
grep = /app/.env.example | sed -e 's/^#\([^ ]\)/\1/' | grep -v -e '^#' | cut -d= -f1 | \
  while read var ; do
    eval "echo \"$var=\\\"\${HUGINN_$var:-\$$var}\\\"\""
  done | grep -v -e ^= > /app/.env
```

It looks for any HUGGINN_ prefixed variable and uses that to replace the environment sourced from the .env.example.

Think of it as an environment override.